src/internal/detect.rs

```diff
@@ -25,29 +25,30 @@ pub fn detect() {
     }
 
     // If pacnew files are found, warn the user and prompt to pacdiff
-    if !pacnew.is_empty() {
-        sp.stop_bold("It appears that at least one program you have installed / upgraded has installed a .pacnew/.pacsave config file. These are created when you have modified a program's configuration, and a package upgrade could not automatically merge the new file.");
+    if pacnew.is_empty() {
+        sp.stop_bold("No pacnew files found");
+    } else {
+        sp.stop_bold("It appears that at least one program you have installed / upgraded has installed a .pacnew config file. These are created when you have modified a program's configuration, and a package upgrade could not automatically merge the new file.");
 
         let choice = prompt!(default false, "Would you like to run pacdiff to deal with this? You can always deal with this later by running `sudo pacdiff`");
```

Comment:
Shouldn't this reference the 'ame diff' command instead of pacdiff?

Reply:
Ame diff is simply a few pretty prompts that decide whether to open pacdiff